tests/data_validation/great_expectations/checkpoints/description_validation.yml

```diff
@@ -23,7 +23,7 @@ validations:
   - batch_request:
       datasource_name: newsletter_automation_datasource
       data_connector_name: default_runtime_data_connector_name
-      data_asset_name: newsletter_automation.articles
+      data_asset_name: Automation corner
```

Comment:
This is not for Automation corner alone but all Articles like Past week and current week too.

Reply:
Updated the data_asset_name as All Articles